lib/general.js

```diff
@@ -48,15 +48,18 @@ class NodeVersionCheck extends DoctorCheck {
 }
 checks.push(new NodeVersionCheck());
 
+async function requirePackageCheck (packageName) {
+  try {
+    await node.requirePackage(packageName);
+    return okOptional(`${packageName} is installed.`);
+  } catch (ign) {
+    return nokOptional(`${packageName} cannot be found.`);
```

Comment:
I would still prefer to show the original error here. Some packages just error out after being imported , even though after it looks like they were installed properly (hello opencv4nodejs 👋 )